src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -886,6 +894,19 @@ class ApplicationManagerImpl
   void ForbidStreaming(uint32_t app_id) OVERRIDE;
 
   /**
+   * @brief Called when application completes streaming configuration
+   * @param app_id Streaming application id
+   * @param service_type Streaming service type
+   * @param result true if configuration is successful, false otherwise
+   * @param rejected_params list of rejected parameters' name. Valid
+   *                        only when result is false.
+   */
+  void OnStreamingConfigured(uint32_t app_id,
+                             protocol_handler::ServiceType service_type,
+                             bool result,
+                             std::vector<std::string>& rejected_params);
```

Comment:
Indeed, thanks! Added in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/a145743eb128a7d0b82bd1fb47e18fa7936dfb1f.